plugins/embedding.cc

```diff
@@ -525,6 +527,58 @@ struct EmbeddingDataset::Itl
         return result;
     }
 
+    virtual std::vector<CellValue>
+    getColumnDense(const ColumnPath & column) const override
+    {
+        auto repr = committed();
+        if (!repr->initialized())
+            throw HttpReturnException(400, "Can't get unknown column");
```

Comment:
It's hard to tell.  But assuming no MLDB bugs, it would normally be a user error.